rex/multi_file_resource.py

```diff
@@ -488,6 +489,12 @@ def preload_SAM(cls, h5_source, sites, unscale=True, str_decode=True,
             strings. Setting this to False will speed up the meta data read.
         tech : str, optional
             SAM technology string, by default 'pvwattsv7'
+        time_step_size: int, optional
```

Comment:
What about `time_index_step`? I would want to input `time_step_size='5min'`

Reply:
Using the step size was MUCH simpler as we didn't have to infer the frequency and then convert to a slice. We could easily do this instead or in addition... We could also do this in reV and just feed over the step_size integer. Thoughts?

Reply:
Yeah yeah sorry i just meant for variable name. Size to me implies how big each timestep is. I definitely dont think we need to input the time step size string.

Reply:
I see, I like time_index_step, i'll change that now